.github/workflows/key4hep_linux.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           cd build
           cmake .. -DCMAKE_CXX_STANDARD=17 \
             -DCMAKE_INSTALL_PREFIX=../install \
+            -DUSE_EXTERNAL_CATCH2=OFF \
```

Comment:
I'd keep that on, I'll put a direct catch2 dependency in `key4hep-stack` to make sure it's there